docs/v8-migration-api.md

```diff
@@ -942,6 +942,16 @@ Most of the changes to our TypeScript-specific AST nodes are to reduce the diffe
 
   __Migration__: Call `t.identifier` with a valid name.
 
+- Reject invalid variable declarator in `t.variableDeclaration` builder ([#10917](https://github.com/babel/babel/pull/10917), [#17217](https://github.com/babel/babel/pull/17217))
```

Comment:
`#10917` was copied accidentally?

Reply:
Nope. It is intentional. This feature was first introduced in #10917: https://github.com/babel/babel/pull/10917/files#diff-64d1d669cf9d0cd3344a526b186c9f7496ac14237a47573926ea505c4bcc8df3R1024